grakn-factory/titan-factory/src/main/java/ai/grakn/factory/TitanPreviousPropertyStepStrategy.java

```diff
@@ -56,9 +56,9 @@
     public void apply(Traversal.Admin<?, ?> traversal) {
 
         // Retrieve all graph (`V()`) steps - this is the step the strategy should replace
-        List<TitanGraphStep> graphSteps = TraversalHelper.getStepsOfClass(TitanGraphStep.class, traversal);
+        List<GraphStep> graphSteps = TraversalHelper.getStepsOfAssignableClass(GraphStep.class, traversal);
```

Comment:
This kind of makes sense. Might have been ignoring the non Titan steps. I am still not sure why this led to the random failures.